app/pages/expenses/travelAndAccommodation/VehicleExpensesPage.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.expenses.travelAndAccommodation
+
+import models.common.BusinessId
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+case object VehicleExpensesPage extends QuestionPage[BigDecimal] {
+
+  override def toString: String = "vehicleExpensesController"
+
+  override def path(businessId: Option[BusinessId]): JsPath = JsPath \ toString
```

Comment:
We need to use the businessId in the path while persisting the data.  We can extend the trait OneQuestionPage and remove the method override def path(businessId: Option[BusinessId])

Reply:
removed the path